instrumentation/opentelemetry-instrumentation-elasticsearch/tests/test_elasticsearch.py

```diff
@@ -50,6 +50,24 @@
 # pylint: disable=too-many-public-methods
 
 
+def normalize_arguments(doc_type, body=None):
+    if major_version == 7:
+        return {"document": body} if body else {}
+    else:
+        return (
+            {"body": body, "doc_type": doc_type}
+            if body
+            else {"doc_type": doc_type}
+        )
```

Comment:
[Recent lint](https://github.com/open-telemetry/opentelemetry-python-contrib/actions/runs/8823753979/job/24224892232?pr=2431) suggests this is an unnecessary `else:

```suggestion
    return (
        {"body": body, "doc_type": doc_type}
        if body
        else {"doc_type": doc_type}
    )
```